### PR TITLE
Some User fields are optional

### PR DESCRIPTION
**Pull request by ericbrandwein**
When the user requested in user_info is an unranked user, the
`rank`, `max rank`, `rating` and `max_rating` fields will not be present. As an example, https://codeforces.com/profile/0ahmad0makki0 is unranked at the time of writing, and the object returned by https://codeforces.com/api/user.info?handles=0ahmad0makki0 hasn't got a `rank`.

### CODE REVIEW

**Comment by soon** [comment]
Thank you for the PR! Could you please update docstrings (where needed) for changed methods? Also please update corresponding tests.

**Comment by ericbrandwein** [comment]
If there's another thing that needs reviewing, don't hesitate on saying.

File: codeforces/api/json_objects/user.py [hunk 1]
```diff
@@ -288,7 +295,7 @@ def rank(self, value):
         :param value: Rank
         :type value: str
```
[the remark this answers is not in the document]
Comment (ericbrandwein) [reply codeforces/api/json_objects/user.py]: Done: https://github.com/soon/CodeforcesAPI/pull/5/commits/fe06673eb15e3cb310540a29019ca57ed5501087

File: tests/api/user_tests.py [hunk 2]
```diff
@@ -94,6 +82,10 @@ def test_load_only_required_from_dict(self):
         self.assertIsNone(self.user.country)
         self.assertIsNone(self.user.city)
         self.assertIsNone(self.user.organization)
+        self.assertIsNone(self.user.rank)
+        self.assertIsNone(self.user.max_rank)
+        self.assertEqual(self.user.rating, 0)
```
[the remark this answers is not in the document]
Comment (ericbrandwein) [reply tests/api/user_tests.py]: Done: https://github.com/soon/CodeforcesAPI/pull/5/commits/fdf8200d745fb66150c25911627d1e87454df200

**Comment by ericbrandwein** [comment]
Great, thanks!

**Pull request merged**